extensions/wikia/CategoryGalleries/CategoryGalleriesHelper.class.php

```diff
@@ -62,7 +62,7 @@ static public function onLanguageGetMagic(&$magicWords, $langCode){
 		/**
 		 * Hook entry for removing the magic words from displayed text
 		 */
-		static public function onInternalParseBeforeLinks(&$parser, &$text, &$strip_state) {
+		static public function onInternalParseBeforeLinks($parser, &$text, &$strip_state) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$parser". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$strip_state". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)